app/assets/javascripts/components/govspeak-editor.js

```diff
@@ -38,6 +38,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var previewMode = previewToggle.innerText === 'Preview'
 
       previewToggle.innerText = previewMode ? 'Back to edit' : 'Preview'
+      previewToggle.ariaLabel = previewMode ? 'Edit ' + textarea.labels[0].innerText : 'Preview ' + textarea.labels[0].innerText
```

Comment:
```suggestion
      previewToggle.ariaLabel = (previewMode ? 'Edit ': 'Preview ') + textarea.labels[0].innerText
```

Very minor suggestion